constants/serverPackets.py

```diff
@@ -279,26 +281,26 @@ def sendMessage(fro: str, to: str, message: str, fro_id: int = 0) -> bytes:
     )
 
 
-def targetBlockingDMs(to: str, fro: str, fro_id: int = 0) -> bytes:
+def targetBlockingDMs(target: str) -> bytes:
     return packetHelper.buildPacket(
         packetIDs.server_targetBlockingNonFriendsDM,
         (
-            (fro, dataTypes.STRING),
             ("", dataTypes.STRING),
-            (to, dataTypes.STRING),
-            (fro_id or userUtils.getID(fro), dataTypes.SINT32),
+            ("", dataTypes.STRING),
+            (target, dataTypes.STRING),
+            (0, dataTypes.SINT32),
```

Comment:
this looks like it's a runtime behavior change:

![image](https://i.cmyui.xyz/kYmOuV_7HbFzd-0cNnU7.png)

is this a "bancho behaves this way" type of change? or something else?

Reply:
hmm i see your comment on james' review, and yeah it is indeed ignored on b.py:

https://github.com/osuAkatsuki/bancho.py/blob/9f96de566fd9c901a30a660bad0197d52319dfe6/app/packets.py#L1237-L1238